packages/atomic/src/themes/coveo.css

```diff
@@ -4,6 +4,7 @@
   --atomic-primary-light: #399ffe;
   --atomic-primary-dark: #1a50ad;
   --atomic-on-primary: #ffffff;
+  --atomic-ring-primary: rgba(19, 114, 236, 0.5);
```

Comment:
I've taken the primary color & set an opacity 50% using rgba, unfortunately we cannot change the opacity of a CSS variable like it was a normal property, hence the new color